### PR TITLE
BAN-L0004: Added Link Component

### PR DESCRIPTION
**Pull request by apyrophob**

### CODE REVIEW

File: src/components/Link/Link.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const LinkComponent = props => {
+    const { linkHref } = props;
+
+    return (
+        <link className='link' href={linkHref} />
```
Comment (likemoody) [inline src/components/Link/Link.js:8]: Let's use tag `<a>`. `<link>` is for something else.

File: src/components/Link/Link.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    );
+};
+
+LinkComponent.PropTypes = {
+    linkHref: PropTypes.string.isRequired,
+};
+
+LinkComponent.defautProps = {
```
Comment (likemoody) [inline src/components/Link/Link.js:16]: Let's remove default props. // remid to talk about that.

File: src/components/Link/Link.js (hunk 1 above)
Comment (apyrophob) [reply src/components/Link/Link.js:8]: We can also use <Link /> component from React.

File: src/components/Link/Link.js [hunk 3]
```diff
@@ -5,14 +5,10 @@ export const LinkComponent = props => {
     const { linkHref } = props;
 
     return (
-        <link className='link' href={linkHref} />
+        <a className='link' href={linkHref}></a>
```
Comment (likemoody) [inline src/components/Link/Link.js:8]: Link has to have title, add `linkTitle` prop here.

File: src/components/Link/Link.js (hunk 1 above)
Comment (likemoody) [reply src/components/Link/Link.js]: Indeed, but we can't use component in itself yet.

File: src/components/Image/Image.js [hunk 4]
```diff
@@ -9,7 +9,7 @@ export const ImageComponent = props => {
     );
 };
 
-ImageComponent.PropTypes = {
```
Comment (likemoody) [inline src/components/Image/Image.js:12]: Warum? (Why?)

File: src/components/Link/Link.js [hunk 5]
```diff
@@ -2,13 +2,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 export const LinkComponent = props => {
-    const { linkHref } = props;
+    const { linkHref, linkTitle } = props;
 
     return (
-        <a className='link' href={linkHref}></a>
```
Comment (likemoody) [inline src/components/Link/Link.js]: Do not forget indentation.

**Review by likemoody — CHANGES_REQUESTED** [review]

File: src/components/Link/Link.js [hunk 6]
```diff
@@ -2,13 +2,14 @@ import React from 'react';
...
+    <a className='link' href={linkHref}>{linkTitle}</a>
```
Comment (likemoody) [inline src/components/Link/Link.js:8]: Do not forget indentation.

**Review by likemoody — CHANGES_REQUESTED** [review]

File: src/components/Image/Image.js [hunk 7]
```diff
@@ -9,7 +9,7 @@ export const ImageComponent = props => {
...
+ImageComponent.propTypes = {
```
Comment (likemoody) [inline src/components/Image/Image.js:12]: Warum? (Why?)

Comment (apyrophob) [reply src/components/Image/Image.js:12]: It works in that way, if you type .PropTypes(not .propTypes) it will not work.

**Comment by likemoody** [comment]
OK, let's merge

**Pull request merged**